Python/Product/Analysis/LanguageServer/Server.cs

```diff
@@ -245,6 +252,14 @@ private sealed class ReloadModulesQueueItem : IAnalyzable {
                 _queue.Enqueue(entry.Value.ProjectEntry, AnalysisPriority.Normal);
             }
         }
+
+        public override Task<object> ExecuteCommand(ExecuteCommandParams @params) {
```

Comment:
I'd really like to pass `async` to the `Command` method. Unfortunately, events don't work with async. Who is going to use `OnCommand`?

Reply:
Extension that needs to receive command upon completion. I.e. the `Command` field in the `CompletionItem`